packages/lib/bookingSuccessRedirect.ts

```diff
@@ -36,9 +36,25 @@ export const getBookingRedirectExtraParams = (booking: SuccessRedirectBookingTyp
     "location",
   ];
 
-  return (Object.keys(booking) as BookingResponseKey[])
+  const basicParams = (Object.keys(booking) as BookingResponseKey[])
     .filter((key) => redirectQueryParamKeys.includes(key))
     .reduce((obj, key) => ({ ...obj, [key]: booking[key] }), {});
+
+  const responseParams = booking.responses
+    ? Object.keys(booking.responses).reduce((obj, key) => {
+        if (Array.isArray(booking.responses[key])) {
+          obj[`responses.${key}`] = booking.responses[key].join(",");
+        } else {
+          obj[`responses.${key}`] = booking.responses[key];
+        }
+        return obj;
+      }, {} as Record<string, never>)
+    : {};
+
+  return {
+    ...basicParams,
+    ...responseParams,
+  };
```

Comment:
It works fine, but looks like type error is visible. Pls fix that

Reply:
Do you want me to do this 
```
const responseParams = booking.responses
  ? Object.keys(booking.responses).reduce((obj, key) => {
      if (Array.isArray(booking.responses[key])) {
        obj[`responses.${key}`] = booking.responses[key].join(",");
      } else {
        obj[`responses.${key}`] = booking.responses[key] as string;
      }
      return obj;
    }, {} as Record<string, string>)
  : {};
  ```

Reply:
It's ok, currently booking response is unknown type, as the type "responses" is not included in the union type of the constraints specified.

Reply:
if it looks good can you approve the PR. Also do I need to get it approved someone as well ?

Reply:
No you don't approve if all the checks are not passed. The type error visible globally, pls fix that